src/show/pages/public/Home/index.jsx

```diff
@@ -1,12 +1,22 @@
 import axios from 'axios';
 import { useEffect, useState } from 'react';
 
-import { ConstructorCard, DriverCard } from '../../../components';
+import {
+  ConstructorCard,
+  DriverCard,
+  UpcomingEvent
+} from '../../../components';
 
 const Home = () => {
   const [constructors, setConstructors] = useState([]);
+  const [event, setEvent] = useState([]);
 
   useEffect(() => {
+    axios
+      .get('http://localhost:3000/v1/events/1')
```

Comment:
```suggestion
      .get('http://localhost:3000/v1/events/does-not-matter')
```
So you know this does not matter, but it has to have someting after `/`. We are going to change this later on with the proper way.

Reply:
So here for now I put just `does-not-matter` ?

Reply:
Yep